tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/preloading/TerminatorWindows.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.ballerina.plugins.idea.preloading;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Launcher Terminator Implementation for Windows. ( Xp professional SP2++).
+ */
+public class TerminatorWindows implements Terminator {
+    private final String processIdentifier = "org.ballerinalang.langserver.launchers.stdio.Main";
+    private static final Logger LOGGER = LoggerFactory.getLogger(TerminatorWindows.class);
+
+    /**
+     * @return file process command.
+     */
+    private String getFindProcessCommand() {
+        // escape forward slashes
```

Comment:
Shall we update all these comments and todos so that they follow the current convention (space after `//` and first letter uppercase).